ng-dev/pr/common/labels/managed.ts

```diff
@@ -32,4 +32,14 @@ export const managedLabels = createTypedObject<ManagedLabel>()({
     name: 'area: build & ci',
     commitCheck: (c: Commit) => c.type === 'build' || c.type === 'ci',
   },
+  DETECTED_PERF_CHANGE: {
+    description: 'Issues related to performance',
+    name: 'area: performance',
+    commitCheck: (c: Commit) => c.type === 'perf',
+  },
+  DETECTED_ADEV_CHANGE: {
+    description: 'Angular.dev documentation',
+    name: 'area: adev',
+    commitCheck: (c: Commit) => c.type === 'adev',
```

Comment:
`adev` is not a commit type right?

Reply:
Ah, yeah. Copy paste since I added all of the area labels initially. Removed.